src/frontend/komponenter/Felleskomponenter/HeaderMedSøk/FagsakDeltagerSøk.tsx

```diff
@@ -24,6 +31,18 @@ import type { IFagsakDeltager, ISøkParam } from '../../../typer/fagsakdeltager'
 import { fagsakdeltagerRoller } from '../../../typer/fagsakdeltager';
 import OpprettFagsakModal from './OpprettFagsakModal';
 
+const ikoner = {
```

Comment:
Dette er en midlertidig løsning for å få riktig typing av ikonene. Disse kommer snart til å bli levert fra designsystemet ifølge Gunn